src/multipolygon/RubberbandMultipolygon.js

```diff
@@ -0,0 +1,137 @@
+import { Selection } from '@recogito/annotorious/src/tools/Tool';
+import { toSVGTarget } from '@recogito/annotorious/src/selectors/EmbeddedSVG';
+import { SVG_NAMESPACE } from '@recogito/annotorious/src/util/SVG';
+import Mask from './PolygonMask';
```

Comment:
This is wrong! should be: import Mask from '@recogito/annotorious/src/tools/PolygonMask';